cpp/src/Slice/Util.h

```diff
@@ -66,7 +66,10 @@ namespace Slice
 
     bool isProxyType(const TypePtr& type);
 
+    // Is this the first element defined in its container?
+    bool isFirstElement(const ContainedPtr& p);
+
     // Is this the first operation defined in the interface?
-    bool isFirstOperation(const OperationPtr& p);
+    inline bool isFirstOperation(const OperationPtr& p) { return isFirstElement(p); }
```

Comment:
Should we really even keep this, instead of just calling `isFirstElement` directly?
The only things interfaces can contain are `Operation`s.

Having both, subtly implies you could have an operation which is the first 'operation' in an interface, but not the first 'element'.

Reply:
Fixed.